apps/custom-ui-floating-window-demo/components/ContactList.js

```diff
@@ -0,0 +1,61 @@
+import { useRouter } from "next/router";
+import { useEffect, useState } from "react";
+import { startOutgoingCall } from "../shared/socket";
+import Footer from "./Footer";
+
+export default function ContactList(context) {
```

Comment:
in some places you are using arrow function for React components and in some places you are using the old function declaration, can you use arrow function in all places?

```suggestion
const ContactList = (context) => {
```